services/openvpn/service/auth_handler_test.go

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2017 The "MysteriumNetwork/node" Authors.
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
```

Comment:
How did you decided yo change this from 2017 to 2018? Was 2017 incorrect year?

Reply:
by accident